src/views/HomeView.vue

```diff
@@ -8,24 +8,17 @@
 <script>
 import Vue from 'vue'
 import router from '@/router'
-import { mapState } from 'vuex'
+import { mapGetters } from 'vuex'
 import SpinnerAnimation from '@/components/animations/SpinnerAnimation.vue'
 
 export default Vue.extend({
   name: 'HomeView',
   components: { SpinnerAnimation },
   computed: {
-    ...mapState(['isContextLoaded', 'isSignedIn'])
+    ...mapGetters(['isSignedIn'])
   },
-  watch: {
-    isContextLoaded: {
-      handler: function (newValue, oldValue) {
-        if (newValue) {
-          router.push({ name: this.isSignedIn ? 'orders' : 'shop' })
-        }
```

Comment:
home view is no longer needed if we are waiting for the context to load before starting the App right ( perf hit ) if it is still needed then ( is the router loaded before the app ? ) then is do not see the point.

Reply:
Yes, this one can go, not sure if we want it in same PR (do we?)

Reply:
Let's do this in new PR